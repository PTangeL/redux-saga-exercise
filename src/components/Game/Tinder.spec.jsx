import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Tinder from './Tinder'

describe('<Tinder />', () => {
  const props = {
    beer: {},
    like: () => {},
    dislike: () => {},
    counterAnimation: ''
  }
  it('should render component with props', () => {
    const wrapper = shallow(<Tinder {...props} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
