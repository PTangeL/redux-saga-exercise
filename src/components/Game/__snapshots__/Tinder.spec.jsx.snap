// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tinder /> should render component with props 1`] = `
<div>
  <BeerContainer
    beer={Object {}}
    counterAnimation=""
  />
  <Actions
    dislike={[Function]}
    like={[Function]}
  />
</div>
`;
